op-proposer/bindings/l2outputoracle.go

```diff
@@ -163,11 +171,11 @@ func NewL2OutputOracleFilterer(address common.Address, filterer bind.ContractFil
 
 // bindL2OutputOracle binds a generic wrapper to an already deployed contract.
 func bindL2OutputOracle(address common.Address, caller bind.ContractCaller, transactor bind.ContractTransactor, filterer bind.ContractFilterer) (*bind.BoundContract, error) {
-	parsed, err := abi.JSON(strings.NewReader(L2OutputOracleABI))
+	parsed, err := L2OutputOracleMetaData.GetAbi()
```

Comment:
where is `L2OutputOracleMetaData` defined? why cannot just use L2OutputOracleABI?

also should we remove `L2OutputOracleABI` then？

Reply:
`L2OutputOracleMetaData` from the generated go bindings file. Also, for the generated bindings, using `GetAbi()` is usually the preferred method.

Reply:
discussed offline. our design principle is to minimize changes to the OP codebase. but given this is generated code, it's fine here